index.html

```diff
@@ -1,12 +1,476 @@
+                                more</a></button>
+                        </div>
+                    </div>
+                    </div>
+                </div>
+                
+            </div>
+        </section>
+
+        <section class="carousel">
+
+            <div class="container">
+                <div class="content border-carousel">
+                    <div class="content-title">
+                        <h2>What Clients Are Saying</h2>
+                    </div>
+
+                    <div class="slider">
+                        <div class="flex-column-align-center slide">
+                            <div class="column-desc">
+                                <p>"Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+                                    been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                                    galley of type."</p>
+                            </div>
+                            <div class="column-img">
+                                <img src="./img/user.png" />
+    
+                            </div>
+                            <div class="column-full-name">
+                                <h3 class="name">William Todd</h3>
+                            </div>
+                            <div class="column-job">
+                                <p>General Counsel and CCO</p>
+                            </div>
+                        </div>
+                        <div class="flex-column-align-center slide">
+                            <div class="column-desc">
+                                <p>
+                                    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+                                been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                                galley of type."
+                                "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
+                                been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                                galley of type."
+                                </p>
+                            </div>
+                            <div class="column-img">
+                                <img src="./img/user.png" />
+    
+                            </div>
+                            <div class="column-full-name">
+                                <h3 class="name">Mark Todd</h3>
+                            </div>
+                            <div class="column-job">
+                                <p>General Counsel and CCO</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
```

Comment:
Go through the whole file and fix all these weird spacings.  Keep formatting in files as clean as possible.